docs/sql-ref-syntax-qry-window.md

```diff
@@ -19,4 +19,222 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+
+Similarly to aggregate functions, window functions operate on a group of rows. However, unlike aggregate functions, window functions perform aggregation without reducing, calculating a return value for each row in the group. Window functions are useful for processing tasks such as calculating a moving average, computing a cumulative, or accessing the value of rows given the relative position of the current row. Spark SQL supports three types of window functions:
```

Comment:
How about `Similarly to aggregate functions, window functions operate on a group of rows.` -> `A window function operates on a group of rows and this is comparable to aggregate functions. `?

Reply:
`window functions perform aggregation without reducing, calculating a return value for each row in the group.` is not clear. This means `window functions do not compute a single aggregated value. Instead, they can generate multiple aggregated values for each group`?